### ISSUE TITLE
[Insight] Files should not be executable

### ISSUE THREAD

**Issue by Ma27** [Ma27/SenNetwork#73]
> Your project contains files with permissive permissions. In order to avoid opening a security breach, you should restrict execution rights on following files: 
- src/Sententiaregum/WebBundle/Controller/GenericTemplateController.php
- src/Sententiaregum/WebBundle/DependencyInjection/SententiaregumWebExtension.php
- src/Sententiaregum/WebBundle/Resources/config/templating.xml
- src/Sententiaregum/WebBundle/Service/AngularPartialService.php

```bash
chmod a-x 'src/Sententiaregum/WebBundle/Controller/GenericTemplateController.php' \
    'src/Sententiaregum/WebBundle/DependencyInjection/SententiaregumWebExtension.php' \
    'src/Sententiaregum/WebBundle/Resources/config/templating.xml' \
    'src/Sententiaregum/WebBundle/Service/AngularPartialService.php'
```

---------------------

_Posted from [SensioLabsInsight](https://insight.sensiolabs.com/projects/cbf672e8-94ee-4817-b59b-0723dcbcce37/analyses/355)_

**Comment by Ma27** [comment]
If there's enough time at weekend, I'll fix the files

-----
The SententiaregumWebExtension is the only affected one since in my refactorization all the other mentioned files will be removed

**Comment by Ma27** [comment]
well I don't know why this commit is still referenced here, I've fixed the wrong message using *git commit --amend*

**Comment by Ma27** [comment]
fixed in 5956bc533800a3bdf1495aa88d898fd483be1017
